index.bs

```diff
@@ -0,0 +1,126 @@
+<pre class="metadata">
+Title: Ambient Light Sensor
+Status: ED
+Level: 2
+ED: https://w3c.github.io/ambient-light/
+Shortname: ambient-light
+TR: http://www.w3.org/TR/ambient-light/
+Previous Version: http://www.w3.org/TR/2015/WD-ambient-light-20150903/
+Editor: Anssi Kostiainen, Intel Corporation, http://intel.com/
+Editor: Tobie Langel 78102, Intel Corporation, http://intel.com/, tobie@sensors.codespeaks.com (from December 2015)
+Editor: Doug Turner, Mozilla Corporation, http://mozilla.com/ (until January 2013)
+Group: dap
+Abstract:
+  This specification defines a concrete sensor interface to monitor
+  the ambient light level or illuminance of the device's environment.
+Version History: https://github.com/w3c/ambient-light/commits/gh-pages/index.bs
+!Bug Reports: <a href="https://www.github.com/w3c/ambient-light/issues/new">via the w3c/ambient-light repository on GitHub</a>
+Indent: 2
+Repository: w3c/ambient-light
+Markup Shorthands: markdown on
+Inline Github Issues: true
+!Issue Tracking: <a href="https://github.com/w3c/ambient-light/milestones/Level%202">Level 2 Issues</a>
+Boilerplate: omit issues-index
+</pre>
+<pre class="anchors">
+
+</pre>
+
+Introduction {#intro}
+============
+
+Intro.
+
+Security and Privacy Considerations {#security-and-privacy}
+===================================
+
+There are no specific security and privacy considerations
+beyond those describe in the Generic Sensor API [[!GENERIC-SENSOR]].
+
+Concepts {#concepts}
+========
+
+The <dfn>AmbientLightSensor</dfn> interface provides information about the
```

Comment:
This ends up being a plain "dfn"-type definition, intended for English terms, rather than the interface definition you probably want.  Since this *isnt'* an interface definition, the IDL block is currently generating a dfn around the interface name as well. As we discussed in IRC earlier, you probably want this to just be a link instead.  `{{AmbientLightSensor}}` is the shorthand syntax for an IDL link.